src/test/scripts/get_product_lts_version.py

```diff
@@ -0,0 +1,61 @@
+import sys
+import urllib.request
+import json
+
+known_lts_version = {
+	'jira-software': '8.13.8',
+	'confluence': '7.12.2',
+	'stash': '7.12.1',
+}
+
+
+def get_lts_version(argv):
+	product = argv[0].lower()
+	if product == 'jira':
+		product = 'jira-software'
+	elif product == 'bitbucket':
+		product = 'stash'
+
+	if product in known_lts_version:
+		url = f"https://my.atlassian.com/download/feeds/archived/{product}.json"
+
+		try:
+			fl = urllib.request.urlopen(url)
+			fdata = fl.read()
+			jsdata = json.loads(fdata[10:len(fdata)-1].decode("utf-8"))
+			enterprise_edition = [x for x in jsdata if x['edition'].lower() == 'enterprise']
```

Comment:
Based on the previous comments, and to make it clear, it might be nice to change this variable name to `lts_edition`

Reply:
changed to `lts_versions`